frontend/public/components/storage-class-form.tsx

```diff
@@ -979,6 +997,26 @@ export class StorageClassForm_ extends React.Component<
               claim is deleted. Defaults to &lsquo;Delete&rsquo;
             </span>
           </div>
+          <div className="form-group">
+            <label className="control-label co-required" htmlFor="expansion-switch">
+              Allow Expansion
+            </label>
+            <div>
+              <Switch
+                className="co-storage-class__switch"
+                id="expansion-switch"
+                label="Enabled"
+                labelOff={expansionLabel}
+                isChecked={isExpansion}
+                onChange={(event) => this.setStorageHandler('expansion', event)}
+                isDisabled={expansionDisabled}
```

Comment:
If expansion is disabled for provisioner we can hide the switch itself.

Reply:
ack